tagmanager/src/tm_workspace.c

```diff
@@ -72,6 +72,9 @@ static gboolean tm_create_workspace(void)
 	theWorkspace->source_files = g_ptr_array_new();
 	theWorkspace->typename_array = g_ptr_array_new();
 	theWorkspace->global_typename_array = g_ptr_array_new();
+
+	tm_source_file_ctags_init();
```

Comment:
Yes, will fix. Originally I didn't have the ctags wrapper file and though all the ctags code would be handled by tm_source_file and this remained.